packages/venia-ui/lib/components/Button/button.js

```diff
@@ -16,7 +16,7 @@ const getRootClassName = priority => `root_${priority}Priority`;
  *
  * @returns {React.Element} A React component that displays a single button.
  */
-export const Button = props => {
```

Comment:
Be careful here -- this was a named export vs the default below so _technically_ this is different.

```
import { Button } from './button'; // named
import  Button from './button'; //default
```

Reply:
Yeah, that's understandable (re: breaking change). I checked the repo and didn't see this named import being used anywhere, so it seemed ok.

This was also the only instance of something like `export const ComponentName` in a file where only the default export existed otherwise. There were other cases with multiple named exports but most of them didn't have a default export.